clang/lib/Sema/SemaInit.cpp

```diff
@@ -2537,19 +2555,13 @@ class FieldInitializerValidatorCCC final : public CorrectionCandidateCallback {
 /// actually be initialized.
```

Comment:
The comment above looks like it needs to be updated with `@param`s for `FinishSubobjectInit`, `TopLevelObject` and `InitializedFields`.

Reply:
I removed addition of `InitializedFields` parameter, so probably it makes sense to update the comment comment in a separate patch.